src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -30,37 +33,110 @@ type OnyxTabNavigatorProps = OnyxTabNavigatorOnyxProps &
         tabBar: (props: TabSelectorProps) => React.ReactNode;
 
         screenListeners?: ScreenListeners<NavigationState, MaterialTopTabNavigationEventMap>;
+
+        /** Callback to register the focus trap container elements of the current active tab */
+        onActiveTabFocusTrapContainerElementChanged?: (containerElement: HTMLElement | null) => void;
+
+        /** Callback to register the focus trap container elements of the tab bar */
```

Comment:
Would you please add a little more detail to these comments? Instead of only describing what the callback are, I like to include when they are triggered and why someone would want to use them. For example:

> This callback is triggered when the container element receives or loses focus. Use this when...